src/routes/formsgSiteLaunch.ts

```diff
@@ -239,6 +249,75 @@ export class FormsgSiteLaunchRouter {
         return null
       })
 
+  private async createMonitor(baseDomain: string) {
+    try {
+      const getResp = await axios.post<{ monitors: { id: string }[] }>(
+        "https://api.uptimerobot.com/v2/getMonitors?format=json",
+        {
+          api_key: config.get("uptimeRobot.apiKey"),
```

Comment:
nit: can this be abstracted out into a constant?

Reply:
Shifted around in https://github.com/isomerpages/isomercms-backend/pull/960/commits/f744c6bf38ae93c9e8f1d4e93ad07cfd0cec8881!